### PR TITLE
German translations

### PR DESCRIPTION
**Pull request by Zekiah-A**

### CODE REVIEW

**Comment by Zekiah-A** [comment]
```xml
    <data name="YouSeemToBeIn" xml:space="preserve">
        <value>You seem to be in</value> <!--TODO: This can't be easily translated, since in mot sentences you say the place, then the verb -->
    </data>
```

Also I am having a bit of issues translating this because it would need to be `Du scheinst in LOCATION zu leben` (zu leben goes after)

German sentence order is more like "You seem in location to live" AFAIK

**Comment by Zekiah-A** [comment]
Also idk if the arabic translation for the word "german" is correct, it is from google translate

File: Athan.Avalonia/Languages/Language.ar.resx [hunk 1]
```diff
@@ -95,6 +95,9 @@
     <data name="Arabic" xml:space="preserve">
         <value>العربية</value>
     </data>
+    <data name="German" xml:space="preserve">
+        <value>الماني</value>
```
Comment (TheVeryStarlk) [inline Athan.Avalonia/Languages/Language.ar.resx:99]: This one is fine, although I suggest "الالمانيه", I don't know if I can edit it from here or not.

**Comment by TheVeryStarlk** [comment]
> ```
>     <data name="YouSeemToBeIn" xml:space="preserve">
>         <value>You seem to be in</value> 
>     </data>
> ```
> 
> Also I am having a bit of issues translating this because it would need to be `Du scheinst in LOCATION zu leben` (zu leben goes after)
> 
> German sentence order is more like "You seem in location to live" AFAIK

It doesn't have to be exactly like the English's version, is there anything close to it in German?

**Pull request merged**